### ISSUE TITLE
Segments are not detected in TXYZ files (but bonds are)

### ISSUE THREAD

**Issue by mquevill** [MDAnalysis/mdanalysis#2440]
**Is your feature request related to a problem? Please describe.**
When reading in a TXYZ file (Tinker `.arc`), the connectivity data is read into the Universe as bonds, but segments are not detected. This means a user must manually break up the segments/residues into manageable chunks. (I have been trying to do this, but have not been able to add segments/residues.)

**Describe the solution you'd like**
Ideally, when a user creates a Universe from a Tinker file, the topology information would detect bonded molecules and create segments/residues for them all.

**Describe alternatives you've considered**
I have tried to break up the segments/residues to more manageable chunks, but I haven't been able to do that successfully.

**Additional context**
Tinker file to test on (rename as single.arc): [single.txt](https://github.com/MDAnalysis/mdanalysis/files/4027178/single.txt)
This file contains 2 molecules: atoms 1-61 are one molecule and atoms 62-65 are another molecule. I would expect this to produce two segments (and maybe 2 residues?) from the detected topology information.
```python
>>> import MDAnalysis as mda
>>> u = mda.Universe("single.arc")
>>> print(len(u.segments))
1
```

**Comment by richardjgowers** [comment]
Hi @mquevill 

I think what you're looking for is called fragments in MDAnalysis.  residues/segments are defined in the topology (txyz in your case) and your file doesn't have this.

Once your bonding is all correct, if you do:
```python
len(u.atoms.fragments)
```
is that what you're looking for?

**Comment by mquevill** [comment]
@richardjgowers You are correct. I was basing it on a code originally written for GROMACS, where every atom is part of a Residue by default. Fragments does the job!
